build/make-distribution.sh

```diff
@@ -220,5 +220,5 @@ TARDIR_NAME="apache-celeborn-$VERSION-$NAME"
 TARDIR="$PROJECT_DIR/$TARDIR_NAME"
 rm -rf "$TARDIR"
 cp -R "$DIST_DIR" "$TARDIR"
-tar czf "apache-celeborn-$VERSION-$NAME.tgz" -C "$PROJECT_DIR" "$TARDIR_NAME"
+tar -cz --no-xattrs --disable-copyfile --exclude='.DS_Store' -f "apache-celeborn-$VERSION-$NAME.tgz" -C "$PROJECT_DIR" "$TARDIR_NAME"
```

Comment:
what's "--disable-copyfile" for? I can not find it in `man tar`

Reply:
do we need `--exclude='.DS_Store'`? it only exists if you opened the folder using `Finder`, I think we don't need that.